modules/demo-magento/shippingMethods/zasilkovna_zasilkovna/shipping.vue

```diff
@@ -37,10 +37,10 @@ const checkout = useCheckout()
 const shipping = useShipping()
 const confirmShippingAddress = useConfirmShippingAddress()
 const confirmShippingMethod = useConfirmShippingMethod()
-
-const picked = computed(() => shipping.shippingAddress)
-
-const address = computed(() => picked.value?.street ?? '')
+const pickedAdressName = ref()
+// const picked = computed(() => shipping.shippingAddress)
+// pickedAdressName need to be changed to maping from state when issue with handlers and dummy contact and init values is resolved
```

Comment:
This looks like a TODO comment. If its that case, start the comment with TODO, otherwise remove it because it doesnt seem useful.